fava/static/javascript/entries.ts

```diff
@@ -24,7 +25,7 @@ abstract class Entry {
   constructor(type: string) {
     this.type = type;
     this.meta = {};
-    this.date = new Date().toISOString().slice(0, 10);
+    this.date = dateToString(new Date());
```

Comment:
There's two other places where this kind of logic is used, could you update those too? And maybe wrap the whole thing up as a `todayAsString()` function